forms/participantdialog.ui

```diff
@@ -183,6 +183,9 @@
    </item>
    <item row="23" column="1">
     <widget class="QCheckBox" name="rosTopicCheckBox">
+     <property name="toolTip">
+      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Only available when a valid ROS 2 installation is detected&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+     </property>
```

Comment:
Question, is it possible to make this appear just if the checkbox is disable? If not, I'm fine with the way it is now.